layouts/partials/webinar-list-item.html

```diff
@@ -1,27 +1,43 @@
-<li class="w-full md:w-1/2 lg:w-1/3 m-0 p-2">
-    <article class="rounded shadow-md bg-white border border-gray-200 mb-10 flex flex-col mx-auto" style="max-width: 320px;">
-        <div style="height: 239px;">
+<li class="w-full m-0 p-2{{ cond (eq .Params.featured true) " order-first" "" }}">
+    {{ if eq .Params.featured true }}
+        <p class="hidden text-2xl font-bold lg:block">FEATURED</p>
+    {{ end }}
+    <article class="rounded shadow-md bg-white border border-gray-200 mb-10 flex flex-col lg:flex-row mx-auto">
+        <div class="w-full {{ cond (eq .Params.featured true) "lg:w-5/12" "lg:w-4/12" }}">
```

Comment:
Having the Featured one larger is OK, but it also sort of makes the ones following it look like they aren't sized/aligned correctly. (When I look at it I wonder "was this by design or a bug?").

<img width="1137" alt="Screen Shot 2020-04-06 at 9 26 24 AM" src="https://user-images.githubusercontent.com/710598/78581407-b457ea80-77e8-11ea-99f7-b2b55c1a5b6c.png">

Also, the FEATURED heading above the featured item, almost makes it seem like it's the heading for all subsequent items. Wondering if we need another heading for the non-featured items or some kind of separator to make it more clear which one is the single featured item, vs the regular items.

Reply:
👍